sagemaker-lineage/sagemaker-lineage.ipynb

```diff
@@ -159,6 +159,8 @@
    "outputs": [],
    "source": [
     "# create an artifact representing inputs to the model building action\n",
+    "# Note that the create artifact API is restricted to SageMaker contexts and will fail if not run from SageMaker Studio for example.\n",
```

Comment:
Change to: Note that the `CreateArtifact` API call is restricted to SageMaker only. For more information please see  [`CreateArtifact`](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateArtifact.html).

Reply:
updated